### DISCUSSION TITLE
Notion Cask signature verification failed

### DISCUSSION THREAD

**Discussion by raymondjxu** [Homebrew/discussions#3243] [Casks]
When I try to install the ``notion`` cask, it complains that Apple hasn't checked it for malware when I try to open it. Downloading from the developer's website presents no such issue.

**Comment by SMillerDev** [comment]
See https://docs.brew.sh/FAQ#why-cant-i-open-a-mac-app-from-an-unidentified-developer

**Reply by raymondjxu** [reply]
The thing is that it doesn’t happen with the download directly from Notion’s website, so it’s probably not a signature thing.
